### ISSUE TITLE
some questions about GroundingDINO with SAM

### ISSUE THREAD

**Issue by lastwhisper-ghy** [IDEA-Research/Grounded-Segment-Anything#390]
Hi . I want to know when you use the box from the GroundingDINO as the prompt, if only boxes greater than the threshold are used as input？

**Comment by rentainhe** [comment]
Yes, we use the output boxes beyond the confidence score as prompts for SAM
